tests/components/Forms/JobForm.test.js

```diff
@@ -0,0 +1,34 @@
+import { render, fireEvent, waitFor } from '@testing-library/react'
+import JobForm from '../../../src/components/Forms/JobForm'
+
+const jobForm = jest.fn().mockImplementation((submit, page) => {
```

Comment:
Please scope this to a test block and add sad path tests for async failures.

Reply:
Please explain what '_Please scope this to a test block_'. is?

'_add sad path tests for async failures_'. Do we need to add a test that catches the modal being displayed on API failure?

Reply:
1. `jobForm` is technically in a global scope prefer function block scope over global so no test mutations arise. 
2. `Do we need to add a test that catches the modal being displayed on API failure?` Ideally, we should capture and test whatever the user experiences.

Reply:
I'm still not sure what you're asking about comment 1.

Comment 2 I'm working on

Reply:
```
describe('Job Search Form', () => {
  const jobForm = jest.fn().mockImplementation((submit, page) => {
    return Promise.resolve({ submit, page })
  })
  test('should set fields and submit form', async () => {
    const { container } = render(<JobForm formData={jobForm} />)
    const form = container.querySelector('form')
    const zipCodeInput = container.querySelector('#zipCode')
    const remoteCheckbox = container.querySelector('#inlineFormCheck')
    const distanceSelect = container.querySelector('#distanceSelect')

    fireEvent.input(zipCodeInput, { target: { value: '02901' } })
    fireEvent.input(remoteCheckbox, { target: { value: 'false' } })
    fireEvent.input(distanceSelect, { target: { value: '40' } })

    expect(zipCodeInput.value).toBe('02901')
    expect(remoteCheckbox.value).toBe('false')
    expect(distanceSelect.value).toBe('40')

    await waitFor(() => fireEvent.submit(form))
    expect(jobForm).toHaveBeenCalled()
  })
  ```